x-pack/plugins/cloud_security_posture/public/pages/vulnerabilities/utils/vulnerabilities_pathname_handler.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiComboBoxOptionOption } from '@elastic/eui';
+import { findingsNavigation } from '../../../common/navigation/constants';
+import { FindingsGroupByKind } from '../../../common/types';
+
+export const vulnerabilitiesPathnameHandler = (
```

Comment:
as i mentioned before, this seems like an overkill to me. 

```typescript
const options = [
   {title: "None", onClick: history.push(vulnerablitiesPage.something.path)},
   {title: "Resrouce", onClick: history.push(vulnerablitiesPage.somethingElse.path)}
]

return 
  <FindingsGroupBySelector options={options} />
```
seems a lot cleaner to me and much easier to add and edit items